src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4554,8 +4554,8 @@ namespace OutputReportTabular {
         //          sourceTypeNames(3)='Gasoline'
         //          sourceTypeNames(4)='Diesel'
         //          sourceTypeNames(5)='Coal'
-        //          sourceTypeNames(6)='FuelOil#1'
-        //          sourceTypeNames(7)='FuelOil#2'
+        //          sourceTypeNames(6)='FuelOilNo1'
+        //          sourceTypeNames(7)='FuelOilNo2'
```

Comment:
Search this source file for lingering \#1 or \#2. Looks like changes are needed at line 9758, 9779, and a few more.

Reply:
Done